libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -32,6 +36,10 @@ const dashboardTable = [
     },
 ];
 
+const TableValues = ["Direct Sales", "Inside Sales"];
```

Comment:
don't need to declare as list again. we can use `[DIRECT_SALES, INSIDE_SALES]` when need